.htaccess

```diff
@@ -1,240 +1,561 @@
 ############################################
-## GoDaddy specific options
+## Default index file
+
+	DirectoryIndex index.php
+
+############################################
+## Custom error pages
 
-#   Options -MultiViews
+	#ErrorDocument 401 /errors/401.php
+	#ErrorDocument 403 /errors/403.php
+	#ErrorDocument 404 /errors/404.php
+	#ErrorDocument 405 /errors/405.php
+	#ErrorDocument 410 /errors/410.php
+	#ErrorDocument 500 /errors/500.php
+	#ErrorDocument 503 /errors/503.php
 
-## you might also need to add this line to php.ini
-##     cgi.fix_pathinfo = 1
-## if it still doesn't work, rename php.ini to php7.ini
+############################################
+## Prevent character encoding issues from server overrides
+## If you still have issues use the second line instead
+
+	AddDefaultCharset Off
+	#AddDefaultCharset UTF-8
 
 ############################################
-## default index file
+## Uncomment if running in cluster environment
+## http://developer.yahoo.com/performance/rules.html#etags
 
-    DirectoryIndex index.php
+	#FileETag none
 
 ############################################
-## php7 settings
+## GoDaddy specific options
+## You might also need to add this line to php.ini
+##    cgi.fix_pathinfo = 1
+
+	#Options -MultiViews
 
 <IfModule mod_php7.c>
 
 ############################################
-## adjust max execution time
+## PHP7 Settings
+
+############################################
+## Adjust max execution time
 
-    php_value max_execution_time 18000
+	#php_value max_execution_time 18000
```

Comment:
we should change this settings, I don't know why the heck it should be suggested to have 18000 seconds max execution time :-D